website/views.py

```diff
@@ -339,12 +339,18 @@ def resolve_guid(guid, suffix=None):
     elif isinstance(resource, Registration) and (clean_suffix in ('', 'comments', 'links', 'components', 'resources',)) and waffle.flag_is_active(request, features.EMBER_REGISTRIES_DETAIL_PAGE):
         return use_ember_app()
 
+    elif isinstance(resource, Registration) and clean_suffix and clean_suffix.startswith('metadata') and waffle.flag_is_active(request, features.EMBER_REGISTRIES_DETAIL_PAGE):
```

Comment:
Is there anything it would `endswith` that isn't just the word `metadata`, e.g. "`metadata-record`" or "`-list`", etc? If not, seems like this could be DRY'd up with the conditional above

Reply:
@adlius any thoughts on this?

Reply:
I was trying to catch the following cases here

-  `/<registration_guid>/metadata/<cedar-metadata-record-id>`
- `/<registration_guid>/metadata/osf`
- `/<registration_guid>/metadata/add`